CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ Deprecated features or components might be removed in later versions without cha
 
 ### 1.next
 
+### 1.6.0
+
+- Add bird-configwatcher
```

Comment:
Please also add info about https://github.com/openvnf/cgw/pull/57

Reply:
Thanks, added info about rclone indentation and security context.